samples/saga/simple/Core_6/Sample/Program.cs

```diff
@@ -16,23 +14,14 @@ static void Main()
     {
         Console.Title = "Samples.SimpleSaga";
         var endpointConfiguration = new EndpointConfiguration("Samples.SimpleSaga");
-        endpointConfiguration.UseSerialization<JsonSerializer>();
-        endpointConfiguration.EnableInstallers();
-
-        var recoverability = endpointConfiguration.Recoverability();
-        recoverability.DisableLegacyRetriesSatellite();
```

Comment:
I'll pr the core to avoid this for the learning transport so that we don't need this crap here